.ruby-version

```diff
@@ -1 +1 @@
-3.3.4
+ruby-3.3.5
```

Comment:
Why did you decide to go for "ruby-"? `ruby-3.3.5` and `3.3.5` are semantically equivalent, aren't they?

Reply:
> `ruby-3.3.5` and `3.3.5` are semantically equivalent, aren't they?

Yes, they are. 

> Why did you decide to go for "ruby-"? 

Since Rails has version `ruby-3.3.5` as a default, I opted to change to this syntax, too: https://github.com/rails/rails/blob/main/railties/lib/rails/generators/rails/app/templates/ruby-version.tt